app/src/utils/helpers.ts

```diff
@@ -10,4 +10,6 @@ export const convertToMap = <T>(array: T[], fieldKey: string): Map<string, T> =>
   })
 
   return result
-}
\ No newline at end of file
+}
+
+export const formatNumber = (num: number) => new Intl.NumberFormat('en-US').format(num)
```

Comment:
Doesn't have to be on this PR, but one thing I dislike about our codebase is mixing function declarations and arrow functions. Could we just use function declarations? I'd like to keep things consistent.